### PR TITLE
Improve mobile styling

### PR DESCRIPTION
**Pull request by lindskogen**

### CODE REVIEW

**Comment by bark** [comment]
Here you do changes in the voting system before his own election. :), But the pr looks good.

File: src/client/CodeInput.js [hunk 1]
```diff
@@ -45,14 +45,15 @@ var CodeInput = React.createClass({
...
-                <span className="dash">-</span>
-                <input type="text" autoCapitalize="none" autoComplete="off" autoCorrect="off" ref={(c) => this.input[2] = c} value={values[2]} onFocus={this.handleOnFocus} onChange={(event) => this.handleOnChange(2, event)}/>
+                <input type="text" style={{width: maxLen + 'em'}} autoCapitalize="none" autoComplete="off" autoCorrect="off" ref={(c) => this.input[0] = c} value={values[0]} onFocus={this.handleOnFocus} onChange={(event) => this.handleOnChange(0, event)}/>
+                <span className="dash"> - </span>
+                <input type="text" style={{width: maxLen + 'em'}} autoCapitalize="none" autoComplete="off" autoCorrect="off" ref={(c) => this.input[1] = c} value={values[1]} onFocus={this.handleOnFocus} onChange={(event) => this.handleOnChange(1, event)}/>
+                <span className="dash"> - </span>
+                <input type="text" style={{width: maxLen + 'em'}} autoCapitalize="none" autoComplete="off" autoCorrect="off" ref={(c) => this.input[2] = c} value={values[2]} onFocus={this.handleOnFocus} onChange={(event) => this.handleOnChange(2, event)}/>
```
Comment (kirayatail) [inline src/client/CodeInput.js]: Is the width enough? 3em for 3 chars sounds kinda tight?

File: src/client/CodeInput.js [hunk 2]
```diff
@@ -45,14 +45,15 @@ var CodeInput = React.createClass({
...
+        const { values } = this.state;
+        const { maxLen } = this.props;
         return (
             <div className="multi-input">
-                <input type="text" autoCapitalize="none" autoComplete="off" autoCorrect="off" ref={(c) => this.input[0] = c} value={values[0]} onFocus={this.handleOnFocus} onChange={(event) => this.handleOnChange(0, event)}/>
-                <span className="dash">-</span>
-                <input type="text" autoCapitalize="none" autoComplete="off" autoCorrect="off" ref={(c) => this.input[1] = c} value={values[1]} onFocus={this.handleOnFocus} onChange={(event) => this.handleOnChange(1, event)}/>
-                <span className="dash">-</span>
-                <input type="text" autoCapitalize="none" autoComplete="off" autoCorrect="off" ref={(c) => this.input[2] = c} value={values[2]} onFocus={this.handleOnFocus} onChange={(event) => this.handleOnChange(2, event)}/>
+                <input type="text" style={{width: maxLen + 'em'}} autoCapitalize="none" autoComplete="off" autoCorrect="off" ref={(c) => this.input[0] = c} value={values[0]} onFocus={this.handleOnFocus} onChange={(event) => this.handleOnChange(0, event)}/>
```
Comment (kirayatail) [inline src/client/CodeInput.js]: Not enough to enforce element width. Overridden anyway. (tested on iPhone (5S?|SE) )
screen shot 2016-05-12 at 08 52 34

**Comment by kirayatail** [comment]
Portrait & Landscape mode on iPhone 6, three chars written in the first field in both cases. 

img_0782
img_0781

**Comment by lindskogen** [comment]
@kirayatail try it now!

**Comment by kirayatail** [comment]
Glorious success! Good job! Visual tests for iDevice 4" and 4.7" approved.

**Pull request merged**